src/components/ec-loading/ec-loading.spec.js

```diff
@@ -31,6 +31,18 @@ describe('EcLoading', () => {
     expect(wrapper.element).toMatchSnapshot();
   });
 
+  it('should render the loading with a background white if the transparent prop is set to false', () => {
```

Comment:
How the test verifies that the background is white?

Reply:
Also, this is wrong I was calling it white but is without any background